FinalWithLED/main.asm

```diff
@@ -0,0 +1,110 @@
+;
+; FinalWithLED.asm
+;
+; Created: 11/28/2018 10:25:47 AM
+; Author : Mizia
+;
+
+
+start:
+	ldi r29, 0x00
+	out tcnt0, r29		;clear initial value of clk
+	;set the RGB LED
+	sbi ddra, 1
+	sbi ddra, 2
+	sbi ddra, 3
+    sbi porta, 1
+	sbi porta, 2
+	sbi porta, 3
+	
+	;set sensor and buzzer
+	cbi ddrb, 0		;input sensor
+	sbi ddrb, 1		;ouput sensor
+	sbi ddra, 0		;input buzzer
+	 
+loop1:
+	;send sygnal to the sensor
+	cbi portb, 1
+	call shortDelay
+	sbi portb, 1
+	call notshortDelay
+	cbi portb, 1
+
+loop2:
+	;wait for echo from the sensor 
+	sbic pinb, 0
+	rjmp one
+	rjmp loop2
+
+one:
+	;start clock
+	ldi r29, 0x05		;clock mode

```

Comment:
When setting up some configuration by writing to a register it is a good idea to be more verbose. It is easy to forget what `0x05` actually does. (even though you are writing a comment). Below is a better approach, and this can also be used almost as is when programming a MCU in C:
```
    ldi r29, (1<<cs00) | (1<<cs02)
    out tccr0b, r29
```
`cs00` and `cs02` are the names of the bits in the `tccr0b` register and you can easily look these up in the datasheet if  you need to change the configuration later.